loader/src/main/scala/com/actian/spark_vector/loader/command/VectorTempTable.scala

```diff
@@ -18,7 +18,7 @@ package com.actian.spark_vector.loader.command
 import org.apache.spark.sql.SQLContext
 
 import com.actian.spark_vector.loader.options.UserOptions
-import com.actian.spark_vector.sql.{ TableRef, VectorRelation }
+import com.actian.spark_vector.sql._
```

Comment:
revert to { TableRef, VectorRelation} and add SparkSqlTable 

they're not so many individual imports to use the full ._ package instead, right?